libs/core/src/lib/menu/services/menu.service.ts

```diff
@@ -132,6 +133,18 @@ export class MenuService {
         this._removeActiveSibling(menuItem);
         this.activeNodePath.push(menuNode);
         menuNode.item.setSelected(true);
+        this.sendSelected(menuNode.item)
+    }
+
+    /** @hidden Sends current selected menu item*/
+    sendSelected(menuItem: MenuItemComponent): void {
```

Comment:
I think you mean "set" not "send"?

Reply:
Also I think this could be used programmatically for many reasons so it should not be hidden from the docs (related to my comment about maybe not always selected the first item)

Reply:
I meant send as it will be sending/emitting it to whatever the parent component is